### ISSUE TITLE
Increase usability for label editing

### ISSUE THREAD

**Issue by maarzt** [maarzt/imglib2-labkit#14]
Currently Labkit's concept for label editing is very much like a very simple version of Paintbrush.
The usability provided by this concept is quite low.

I can think of many things, that would simplify label editing:
* Splitting a labels into connected components.
* Selecting connected components (to remove / copy / ... etc).
* Joining two labels into one.
* Split a label into multiple layers like known from Gimp / Photoshop.

**Comment by imagejan** [comment]
Aren't these all features that could (and should) be dealt with on a different level, say [`imglib2-roi`](https://github.com/imglib/imglib2-roi)? It would be awesome if Labkit could just wrap the ROI model from there :-)

**Comment by maarzt** [comment]
That's true. But currently there's not much overlap between imglib2-roi. If think imglib2-roi currently only provides geometric shapes, and there's no editor for imglib2-roi? Labkit currently is an editor for for free shaped labels only.

For me an ideal ImageJ2 region of interest future looks like this:
* BDV becomes the default image displayer in ImageJ2.
* IJ1 rois are replaced by imglib2-roi.
* Labkit becomes part of the default image displayer and is used for imglib2-roi editing.

If there's any effort to build an imglib2-roi editor, I would want to be part of that.
